packages/gatsby-theme-sso-ui-kit/src/components/login-button.tsx

```diff
@@ -30,20 +31,37 @@ const LoginButton = (props: LoginButtonProps) => {
       : window.location.pathname;
   };
 
+  if (props.quizId) {
+    return (
+      <SecondaryButton
```

Comment:
I kept the login button for the quiz so far. I'm not sure if we still want to use it in some way or if we can simply remove it and show something else in the quiz box when the user is not logged in.

Reply:
there is an issue already for this https://github.com/commercetools/commercetools-docs-kit/issues/1620